x/ecocredit/migrations/v3/keys.go

```diff
@@ -92,11 +94,20 @@ func IterateSupplies(store sdk.KVStore, storeKey byte, cb func(denom, supply str
 	return nil
 }
 
-// Calculate the ID to use for a new project, based on the class id and
-// the project sequence number.
-//
-// The initial version has format:
-// <class id><project seq no>
-func FormatProjectID(classID string, projectSeqNo uint64) string {
-	return fmt.Sprintf("%s%02d", classID, projectSeqNo)
+func parseBatchDenom(denom string) (*time.Time, *time.Time, error) {
+	// batch denom format: <class id>-<start date>-<end date>-<batch seq no>
```

Comment:
```suggestion
	// batch denom format: <project id>-<start date>-<end date>-<batch seq no>
```

Reply:
This is the version of the batch denom we are migrating from.